service/Auth.ts

```diff
@@ -99,3 +99,25 @@ module.exports.removeScope = (event, context, callback) => {
     }
   })
 }
+
+module.exports.getUserList = (event, context, callback) => {
+  // check header or url parameters or post parameters for token
+  const full_token = event.authorizationToken
+  if (!full_token) return callback(null, "Unauthorized")
+  const token = full_token.split(" ")[1]
+
+  // verifies secret and checks exp
+  jwt.verify(token, process.env.JWT_SECRET, (err, decoded) => {
+    if (err) return callback(null, "Unauthorized")
+    const {scopes} = decoded
+    if (scopes.includes("root")) {

```

Comment:
Here you could put that it is available for root and "getUserList".

Similar this line
```
    if (scopes.includes("root") || scopes.includes("removeScope")) {
```

This way we don't need to give root access to users we want to able to view user list.

Reply:
It is done :)